timeseries-streaming/timeseries-java-applications/TimeSeriesPipeline/src/test/java/common/TSTestData.java

```diff
@@ -134,6 +134,13 @@ public Builder setInputTSDataFromJSON(
                 if (advanceWatermark_expression == AdvanceWatermarkExpression.INFINITY) {
                   testStream = stream.advanceWatermarkToInfinity();
                 }
+                break;
+              case ADVANCE_PROCESSING_TIME_SECONDS:
```

Comment:
I think the other use case we need is to advance the watermark but then have a event that has timestamp > than the watermark. 

For example:

event{timestamp: 1}
advanceWaterMark timestamp: 3
event{timestamp: 2}